t/test-data/Models/hr-inclNOxExtendedWithFilters/Livestock/Equides/Excretion.nhd

```diff
@@ -293,7 +293,7 @@ Flisch R, Sinaj S, Charles R, Richner W 2009. Grundlagen für die die Düngung i
         writeLog({
             en => "Die eingegebene N-Ausscheidung für $cat weicht mehr als 75% vom Standard ab.",
             de => "Die eingegebene N-Ausscheidung für $cat weicht mehr als 75% vom Standard ab.",
-            fr => "Die eingegebene N-Ausscheidung für $cat weicht mehr als 75% vom Standard ab.",
+            fr => "Die eingegebene N-Ausscheidung für $cat weicht mehr als 75% vom Standard ab."
```

Comment:
Also in Perl/Raku sind trailing commas erlaubt (bzw sogar empfohlen) in einem Hash (was das hinzufügen und umsortieren einfacher macht).